chain/epoch-manager/src/shard_tracker.rs

```diff
@@ -269,6 +266,15 @@ impl ShardTracker {
         self.tracked_shards_config.tracks_all_shards()
     }
 
+    /// Returns whether the node tracks a non-empty subset of shards.
+    pub fn tracks_some_shards(&self) -> bool {
+        match &self.tracked_shards_config {
+            TrackedShardsConfig::AllShards => true,
+            TrackedShardsConfig::Shards(shards) => !shards.is_empty(),
+            _ => false,
```

Comment:
Sorry, this again, why doesn't it return true for the other types of tracked shards config? I think you may want to rename this method or otherwise adjust it. In the current form it limits archival nodes to either AllShards or Shards tracking. I'm not against it though I also don't see reason for this limitation. 

Maybe rename to `is_valid_for_archival` or similar and allow: AllShards, Shards and Accounts.

Reply:
I'll rename it to `is_valid_for_archival`. For SST, I'm currently only interested in `Shards`. I'm not saying it won't work for `Accounts`, but supporting that would require additional verification and testing. So for now, I'll just leave a comment noting the potential for `Accounts` support.